contracts/secretswap_router/src/msg.rs

```diff
@@ -0,0 +1,76 @@
+use cosmwasm_std::{Binary, HumanAddr, Uint128};
+use schemars::JsonSchema;
+use secretswap::Asset;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct InitMsg {}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct Hop {
+    pub from_token: Token,
+    pub pair_address: HumanAddr,
+    pub pair_code_hash: String,
+    pub expected_return: Option<Uint128>,
```

Comment:
If you only care about expected_return once (on the last hop) move it to the Route, instead of in the Hop?